docs/development/pigs/pig-024.rst

```diff
@@ -0,0 +1,220 @@
+
+Given the fact that Gammapy is licensed under a 3-clause BSD style license (see gammapy/LICENSE.rst), Gammapy can be used
+and even modified for a science project. For this modified version, the proposed authorship policy of this PIG is not
+applicable but the general citation scheme should be applied.
+
+This PIG is structured as follows: a reminder of our general citation scheme that was up-to-now only given orally and on
+our web pages; then, the authorship policy is given for each of the products associated with Gammapy, namely the intermediate
+releases, the Long Term Support (LTS) releases, the general Gammapy papers, and the contributions to the conference.
+
+2. Reminder of the citation scheme
+==================================
+
+When Gammapy is used for any publication, contribution to conferences or software, authors should cite properly Gammapy.
+It is asked to cite the DOI of the latest Gammapy paper.
+
+.. note::
+    As the Gammapy community fully supports Open Science, we strongly encourage authors using Gammapy to follow
+    the FAIR4RS principles and to allow the reproducibility of their results. As consequence, we suggest always mentioning
+    the `Zenodo <https://zenodo.org/>`_ DOI or the `HAL <https://hal.archives-ouvertes.fr/>`_ identifier (associated with
+    the `Software Heritage <https://www.softwareheritage.org/>`_ archive) of the used release of Gammapy.
+
+This scheme and these two references are precise on our web pages.
+
+3. Authorship policy
+====================
+
+The Gammapy references contain a list of authors, or `contributors`, that requires to be updated with time according to a
+general policy. This section defines what is a contributor to Gammapy and the policy for each type of product.
+
+Definition of a `Contributor`
+-----------------------------
+
+Contributing to Gammapy can be made in different ways:
+
+- contributing to gammapy source code,
+- contributing to our documentation (rst pages, docstring, gammapy-webpage),
+- contributing to our DevOps environment (CI, GitHub configurations, gammapy-docs, release processes, etc),
+- contributing to our associated projects (gammapy-benchmarks, gammapy-data, gammapy-extra, gammapy-cat, gammapy-recipes),
+- contributing to the maintenance of our repositories (conda-forge, PyPI, Zenodo, HAL, docker-hub, etc) or open hub (openhub, etc),
+- leading the precise tasks identified by the Lead Developers,
+- organizing and leading hands-on sessions or schools,
+- managing our social media accounts,
+- and, de-facto, the members of the Coordination Committee.
+
+Any contribution on these topics renders the user as an official Gammapy contributor.
+
+.. important::
+    We strongly recommend that the contributors use always the same GitHub account, to have a unique
+    `ORCID <https://orcid.org/>`_ identifier and store it into your GitHub user profile, and inform the developers
+    of any change of affiliation.
```

Comment:
Changes in GIthub user names are maintained in https://github.com/gammapy/gammapy/blob/master/.mailmap It would be good specify whether the responsibility of maintenance is on the Gammapy project side or contributor side. The latter would mea:  "if you don't make sure the info is correct, your contributions will not attributed correctly". Maybe it's both...

Reply:
I think the list of contributors should be maintained by gammapy, in a metadata file (many choices: mailmap, citation.cff, codemeta.json, zenodo.json, tributors...) with the responsability falling on contributors to make sure their affiliation is correct, and update it through PR if not.
The CI could check if the author of a PR is already listed in this metadata (prob some dev needed ther)

Reply:
We are mixing 2 thinks, I think...
The list of contributors should indeed manager by Gammapy.
However, the user profile should be managed by them-self. They contain private information and as an open project (and not a closed collaboration), I feel very uncomfortable to deal with users data. This is why I put this important recommendation!